resources/views/list/partials/thead.twig

```diff
@@ -1,39 +1,39 @@
 
-        {% for cF in listingCFs %}
-            {% set isSortable = cF.isSortable %}
-            <th class="text-nowrap{{ isSortable ? ' sortable' }}{{ param.sort_by == 'cf_' ~ cF.id ~ '_desc' ? ' sort-by sort-desc' : (param.sort_by == 'cf_' ~ cF.id ~ '_asc' ? ' sort-by') }}"
-                data-sort-asc="{{ 'cf_' ~ cF.id ~ '_asc' }}" data-sort-desc="{{ 'cf_' ~ cF.id ~ '_desc' }}">
-                {{ cF.name }}
-                {% if isSortable %}
-                    {{ img('visiosoft.module.advs::images/sort-arrow.svg').data|raw }}
-                {% endif %}
-            </th>
-        {% endfor %}
-
-        {% if listLocation %}
-            <th class="text-nowrap sortable{{ param.sort_by == 'address_z_a' ? ' sort-by sort-desc' : (param.sort_by == 'address_a_z' ? ' sort-by') }}"
-                data-sort-asc="address_a_z" data-sort-desc="address_z_a">
-                {% set firstLocationType = setting_value('visiosoft.module.advs::location_data_type_first') %}
-                {% set secondLocationType = setting_value('visiosoft.module.advs::location_data_type_second') %}
-                    {{ trans('visiosoft.module.advs::field.'~firstLocationType~'.name') }}
-                    / {{ trans('visiosoft.module.advs::field.'~secondLocationType~'.name') }}
-                {{ img('visiosoft.module.advs::images/sort-arrow.svg').data|raw }}
-            </th>
-        {% endif %}
-        {% if setting_value('visiosoft.module.advs::market_place') and showDate %}
-            <th class="text-nowrap sortable{{ param.sort_by == 'sort_time_newest' ? ' sort-by sort-desc' : (param.sort_by == 'sort_time_oldest' ? ' sort-by') }}"
-                data-sort-asc="sort_time_oldest" data-sort-desc="sort_time_newest">
-                {{ trans('visiosoft.module.advs::field.date.name') }}
+    {% for cF in listingCFs %}
+        {% set isSortable = cF.isSortable %}
+        <th class="text-nowrap{{ isSortable ? ' sortable' }}{{ param.sort_by == 'cf_' ~ cF.id ~ '_desc' ? ' sort-by sort-desc' : (param.sort_by == 'cf_' ~ cF.id ~ '_asc' ? ' sort-by') }}"
+            data-sort-asc="{{ 'cf_' ~ cF.id ~ '_asc' }}" data-sort-desc="{{ 'cf_' ~ cF.id ~ '_desc' }}">
+            {{ cF.name }}
+            {% if isSortable %}
                 {{ img('visiosoft.module.advs::images/sort-arrow.svg').data|raw }}
-            </th>
-        {% endif %}
-        <th class="text-nowrap sortable{{ param.sort_by == 'sort_price_up' ? ' sort-by sort-desc' : (param.sort_by == 'sort_price_down' ? ' sort-by') }}"
-            data-sort-asc="sort_price_down" data-sort-desc="sort_price_up">
-            {{ trans('visiosoft.module.advs::field.price.name') }}
+            {% endif %}
+        </th>
+    {% endfor %}
+
+    {% if listLocation %}
+        <th class="text-nowrap sortable{{ param.sort_by == 'address_z_a' ? ' sort-by sort-desc' : (param.sort_by == 'address_a_z' ? ' sort-by') }}"
```

Comment:
is it necessary? @erhanask

Reply:
For desktop these are can be necessary, so i just added "d-none" class on mobile for <tr> tag that used inside of <thead> tag . @fatihalp